### ISSUE TITLE
Adding Answers on top bar

### ISSUE THREAD

**Issue by SmartManoj** [soscripted/sox#486]
![image](https://user-images.githubusercontent.com/7231077/102612291-76241080-4156-11eb-8321-33aa9472465c.png)
[Source](https://stackoverflow.com/questions/44520958/uncaught-error-cannot-find-module)
**Use case**
If questions are too big when going through similar questions, it would be handy to check whether it is answered or not.

**Suggested descriptions**
Answers at the top bar.

```
_=$('#answers-header h2 span')[0].firstChild.textContent
$('.grid.fw-wrap')[0].insertAdjacentHTML('beforeEnd',` <div class="grid--cell ws-nowrap mr16 mb8">  <span> Answers</span> ${_}</div>`)

```

**Comment by shu8** [comment]
@SmartManoj Thanks for the feature request, this sounds like a useful feature!

Until it's done -- would using the Stack Exchange keyboard shortcut `J` (to skip to next post) work for you here?

If you press `jj` on your keyboard when you navigate to a post, it will jump to the first answer if it exists!

**Comment by shu8** [comment]
This is now implemented in [v2.7.6 dev](https://github.com/soscripted/sox/raw/dev/sox.user.js), under `Appearance` :)
